commit 26cc1bb38ed4575d42a09f9ba8869513d1341fd1 (refs/changes/22/3622/1)
Author: Jan Wyszynski <jan.wyszynski@joyent.com>
Date:   2018-03-14T00:46:48+00:00 (1 year, 7 months ago)
    
    joyent/node-zkstream#32 ZKBuffer emits unhandled error on packet opcode
    CLOSE_SESSION

diff --git a/lib/zk-buffer.js b/lib/zk-buffer.js
index 51549f4..3556118 100644
--- a/lib/zk-buffer.js
+++ b/lib/zk-buffer.js
@@ -317,6 +317,7 @@ ZKBuffer.prototype.readResponse = function (xidMap) {
 		case 'PING':
 		case 'SYNC':
 		case 'DELETE':
+		case 'CLOSE_SESSION':
 			/*
 			 * No special response packet format -- error code in
 			 * header determines status of the request.
